packages/docs/docs/theme/default-theme-config.md

```diff
@@ -69,14 +69,14 @@ module.exports = {
 }
 ```
 
-Outbound links automatically get `target="_blank" rel="noopener noreferrer"`. You can offer `target` and `rel` to customize the attributes:
+Outbound links automatically get `target="_blank" rel="noopener noreferrer"`. You can offer `target` and `rel` to customize the attributes. Passing a "falsy" value to the `rel` attribute will disable that attribute for a link:
```

Comment:
Oh, maybe I should make this clearer to say that the falsy value can't be `undefined`.

Reply:
IMO, change the rule. Let user set `false` rather than falsy value to disable the attribute.

Reply:
That makes sense.